index.rst

```diff
@@ -326,33 +326,162 @@ Analysis of unbinned data is clearly needed for pipeline data quality analyses,
 Databases
 ^^^^^^^^^
 
-.. warning::
+This section identifies commissioning-specific database needs from a series of `use cases`_.
 
-   This section is not yet completed and only reports the current status.
+We revisit the need for a Visit database at the Summit.
+This relational database consolidates raw image metadata, Rapid Analysis outputs, telemetry data, and other datasets related to the observations.
 
-Data from the observatory will come from numerous sources.
-To minimize maintenance and facilitate ease-of-use, efforts should be made to minimize the number of active databases.
-Suggestions to create new individualized databases should be resisted unless no existing database can be utilized.
-Whereas much of the data coming off the summit is time based and therefore goes into a time-based database (the EFD), other aspects of the system are image based, such as what will be produced by Rapid Analysis and the parts of the camera system.
-The implementation of various project databases is currently being discussed and documented in a number of tech notes [#]_ however, the capabilities and functionalities required by the commissioning team has not been explicitly described.
+We discuss possible paths for its implementation using existing tools in DM, bearing in mind the near real-time feedback requirements of Rapid Analysis.
 
-.. [#] For further details, consult the following technotes, which are in various states of being written: `Sasquatch`_, the `Butler <https://dmtn-204.lsst.io>`_,  database support for `campaigns <https://dmtn-220.lsst.io/>`_, as well as the `consolidated database <https://dmtn-227.lsst.io/>`_.
+In addition to the Visit database, we recommend recording scalar metrics into a time series database for further monitoring and visualization.
```

Comment:
Maybe tone this down a bit.
Consider:
Note that in addition to the visit database, values, such as scalar metrics, can also be added to other databases when it makes sense to do so.